### ISSUE TITLE
MemoryChecker default priority makes PC barely usable

### ISSUE THREAD

**Issue by isidroco** [lordmulder/MemoryChecker#1]
Please make default priority Below Normal instead of Above Normal as it is now, ie: Renaming a file has a huge lag while ram is being tested.

**Comment by lordmulder** [comment]
"Above Normal" is only a very slight priority boost and shouldn't have much impact on other applications.

Anyways, you really **should not** use your PC for anything else while testing RAM!

MemoryChecker allocates almost all memory so that it can be tested. This alone can make the system appear "sluggish". Also, any memory allocated by other applications can **not** be tested. So you want to close **all** other applications before/while testing RAM.

In other words, close **all** applications, start the MemoryChecker and then leave your PC alone until test has completed.

**Comment by isidroco** [comment]
Yes, but on win7 above normal does seriously impact on otherwise trivial tasks. Even launching taskmanager and lower the priority is difficult to do, please try it.  Sometimes I use it to check system stability (ie: memoryChecker 30%), which only test a small portion of memory but stresses CPU and use other stuff to see if system is reliable under stress.  As said, please change default to below normal, or at least give us a command line option to do it (ie: --low ) . Thanks!
